packages/shared/src/types/abTests/banner.ts

```diff
@@ -79,7 +79,7 @@ export const bannerTestFromToolSchema = testSchema.extend({
     locations: z.array(countryGroupIdSchema),
     contextTargeting: pageContextTargetingSchema,
     variants: z.array(bannerVariantFromToolSchema),
-    articlesViewedSettings: articlesViewedSettingsSchema.optional(),
+    articlesViewedSettings: articlesViewedSettingsSchema.optional(),    
```

Comment:
`yarn lint --fix` should fix this type of thing

Reply:
ran and pushed